sites/docs/src/content/docs/guidelines/components/modules.md

```diff
@@ -324,6 +324,73 @@ See the [Bash manual on file operators](https://tldp.org/LDP/abs/html/fto.html)
 
 Alternate suggestions include using `grep -c` to search for a valid string match, or other tool which will appropriately error when the expected output is not successfully created.
 
+### Script inclusion
+
+:::warning
+You SHOULD NOT use [Nextflow module binaries](https://www.nextflow.io/docs/latest/module.html#module-binaries), as these are not fully portable across all execution contexts.
+:::
```

Comment:
I'm not sure if this is the right place to have this warning. IMO - this is documentation of the nf-core way of doing things, and it's not necessary to say we don't use binaries because for whatever reason. It could be rephrased as a positive statement about templates being portable rather than being negative about module binaries.

Reply:
Fair point, will propose a new one,